microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -1,11 +1,11 @@
 ---
```

Comment:
I think this .md is trying to do too deep-nested and complex of page structure.  The PR's current outline is:
```
## Introduction
#### Limitations
#### Related feature: Overrides

## Set up the environment for the tutorial
#### Set up the demo
#### Prepare DevTools for file editing

## Edit and save source code files
#### Save a CSS change to disk
#### Save an HTML change to disk
###### Try changing the HTML from the Elements panel
###### Change HTML from the Sources tool
#### Save a JavaScript change to disk

## Next steps
```

Recommend forcing all headings to h2 (only h2 appears in "In this article" nav), and then demoting headings we want to hide such as near top of article.  Strongly favor h2's as much as possible.

Avoid pattern:
h2 abstract, low-value bucket
h4 high-value heading
h4 high-value heading

Re-express as a simple flat list of h2's, decorated with some low-importance h4's as needed for skimmability within each h2 section.

Reply:
makes sense